esp32h2-hal/Cargo.toml

```diff
@@ -41,7 +41,7 @@ embedded-can       = "0.4.1"
 embedded-io-async  = "0.5.0"
 esp-backtrace      = { version = "0.7.0", features = ["esp32h2", "panic-handler", "exception-handler", "print-uart"] }
 esp-hal-smartled   = { version = "0.4.0", features = ["esp32h2"], path = "../esp-hal-smartled" }
-esp-println        = { version = "0.5.0", features = ["esp32h2"] }
+esp-println        = { version = "0.5.0", features = ["esp32h2", "log"] }
```

Comment:
Not a big deal, but I see the `log` feature is enabled and the logger is initialized in the example, however no logging seems to actually take place (`println!` is used).

I assume this was for debugging?